moped-api/users/users.py

```diff
@@ -117,12 +125,13 @@ def user_create_user(claims: list) -> (Response, int):
                     UserPoolId=USER_POOL,
                     Username=email,
                     TemporaryPassword=password,
+                    MessageAction="SUPPRESS",
```

Comment:
this is what prevents AWS from sending emails upon sign up

Reply:
👀